test/regress/cli/CMakeLists.txt

```diff
@@ -3808,6 +3808,7 @@ set(testers
   unsat-core
   proof
   lfsc
+  alethe
```

Comment:
For now remove this so that the tester cannot be used via cmake. Only when the support for Alethe is stabilized in main we will enable this.